docs/requirements.txt

```diff
@@ -1,7 +1,7 @@
-Sphinx==7.4.7
-sphinx-rtd-theme==2.0.0
-sphinx-autobuild==2024.9.19
-sphinxcontrib-bibtex==2.6.3
+Sphinx
+sphinx-rtd-theme
+sphinx-autobuild
+sphinxcontrib-bibtex
```

Comment:
Any particular reason you removed the version here? Seems reasonable though

Reply:
Those version didnt work for me. But I reverted since we should rather replace them with the newer versions